Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -45,15 +43,23 @@
 	
 	private static final int ADD_EXPENSE_CLAIM_REQUEST = 1;
 	private static final int EDIT_EXPENSE_CLAIM_REQUEST = 2;
+	
 	private static final String EXPENSE_CLAIM_FILENAME = "claims";
 
 	//================================================================================
 	// Properties
 	//================================================================================
 	
-	private ArrayList<ExpenseClaim> claims;
-	private ExpenseClaimArrayAdapter adapter;
+	private static ArrayList<ExpenseClaim> claims;
+	private static ExpenseClaimArrayAdapter adapter;
 	private int longPressedItemIndex;
+	
+	public static ExpenseClaimArrayAdapter getAdapter () {
```

Comment:
Actually, it would be better to return a comparator object from inside the result intent so that you can use the comparator to sort inside `commitClaimsMutations()`.